orttraining/orttraining/core/graph/pipeline_transformer.cc

```diff
@@ -325,7 +671,9 @@ Status TransformGraphForPipeline(
   graph.SetOutputs(outputs_args_sets);
   graph.SetGraphResolveNeeded();
   graph.SetGraphProtoSyncNeeded();
-  return graph.Resolve();
+  graph.Resolve();
```

Comment:
>graph.Resolve(); [](start = 2, length = 16)

graph.Resolve() can return error status. Let's not ignore it.

Reply:
Sure.